docs/Behaviour - Client Side.md

```diff
@@ -1,16 +1,16 @@
 # Behaviour: Client Side
 
-## Constraints of Sender
+## Active Constraints of Sender
 
 Before making an [IS-05][IS-05] connection, NMOS Controller chooses a Sender and Receivers. Then NMOS Controller MUST `GET /constraints/supported` from the Sender, collect Receiver Capabilities from the Receivers and make such intersection of them that they can be utilized for building Constraints satisfying all the Receivers and using Parameter Constraints supported by the Sender. If the Sender supports less Parameter Constraints than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. After that NMOS Controller MUST `PUT /constraints/active` and make the IS-05 connection if the Constraints have been applied succesfully. After breaking this connection via IS-05, NMOS Controller is RECOMMENDED to `DELETE /constraints/active`.
 
-If multiple Receivers have equal Constraint Sets, NMOS Controller MUST take only one of them. If they differ only in `urn:x-nmos:cap:meta:preference`, then one with the highest priority MUST be taken.
+If multiple Receivers have equal Constraint Sets, NMOS Controller MUST choose only one of them while making the intersection. If they differ only in `urn:x-nmos:cap:meta:preference`, then one with the highest priority MUST be taken.
```

Comment:
Now I understand this is a bit better, my question is whether the Controller has to use the preference values from the Receivers at all? The Controller may know that one Receiver is higher priority than another, for example? And in any case, the way `meta:preference` is defined, values can only be compared for the same Receiver, you can't compare between them.

Reply:
```suggestion
```

Reply:
I think I would remove that paragraph ... because computing the intersection is much more complex than that. We could state that urn:x-nmos:cap:meta:label, urn:x-nmos:cap:meta:preference properties of constraint_set(s) MUST be ignored for the purpose of calculating the intersection and those constraint_set(s) with urn:x-nmos:cap:meta:enabled set to false MUST be ignored completely.

Ex. one Receiver indicate  height in range 720 <= height <= 2160 and another indicate height == 1080 ... The intersection is height 1080 but the constraint_set(s) are very different.

Reply:
I'm about to remove it too. Can `urn:x-nmos:cap:meta:label` and `urn:x-nmos:cap:meta:preference` be used for setting `urn:x-nmos:cap:meta:preference` to Constraint Sets of the intersection? An example for `urn:x-nmos:cap:meta:preference` is given in a suggestion below.